cdc/owner/changefeed.go

```diff
@@ -41,7 +41,7 @@ import (
 	"go.uber.org/zap"
 )
 
-// newSchedulerFromCtx creates a new schedulerV2 from context.
+// newSchedulerFromCtx creates a new schedulerV3 from context.
 // This function is factored out to facilitate unit testing.
```

Comment:
after `schedulerV2` removed, there is only one scheduler implementation in the cluster, so no need to name it `schedulerV3`, just name it `scheduler` should be ok?

Reply:
yes. I'll update it. Thanks!